examples/with-sentry/pages/_error.js

```diff
@@ -1,56 +1,82 @@
-import NextErrorComponent from 'next/error'
+/**
+ * NOTE: If you are on `@sentry/nextjs` version 7.2.1 or higher, you can replace
+ * what's below with the simpler version which appears in a comment at the end
+ * of this file.
+ *
+ * This page is loaded by Nextjs:
+ *  - on the server, when data-fetching methods throw or reject
+ *  - on the client, when `getInitialProps` throws or rejects
+ *  - on the client, when a React lifecycle method throws or rejects, and it's
+ *    caught by the built-in Nextjs error boundary
+ *
+ * See:
+ *  - https://nextjs.org/docs/basic-features/data-fetching/overview
+ *  - https://nextjs.org/docs/api-reference/data-fetching/get-initial-props
+ *  - https://reactjs.org/docs/error-boundaries.html
+ */
 
 import * as Sentry from '@sentry/nextjs'
+import NextErrorComponent from 'next/error'
 
-const MyError = ({ statusCode, hasGetInitialPropsRun, err }) => {
-  if (!hasGetInitialPropsRun && err) {
-    // getInitialProps is not called in case of
-    // https://github.com/vercel/next.js/issues/8592. As a workaround, we pass
-    // err via _app.js so it can be captured
-    Sentry.captureException(err)
-    // Flushing is not required in this case as it only happens on the client
+const COMPATIBILITY_ERROR =
```

Comment:
I would move this into `Sentry.captureUnderscoreErrorException`. It is not user's job to handle this error. It's ours.

Reply:
See below.